### DISCUSSION TITLE
PHP 8.1

### DISCUSSION THREAD

**Discussion by skurvish** [trobfab/fabrik#15] [Ideas]
I think we should all be using PHP8.1 for this development and make it a minimum requirement for this release.

**Comment by joomlahenk** [comment]
Agree, but not for now. I already have many issues to solve and changing to 8.1 will cause even more I expect.
If all works we can change to 8.1.

**Comment by trob** [comment]
I'm running php8.1 on my local host test site

**Comment by skurvish** [comment]
As long as @trob and I are running 8.1 we should catch any issue that arise.

**Comment by skurvish** [comment]
Worth keeping this in mind, the appendices contain details of changes between the releases and what to do about them:

https://www.php.net/manual/en/appendices.php

**Comment by skurvish** [comment]
8.1 is now a required version of F4.
